### PR TITLE
Twis37

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @Manishearth

[Review on Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/35)

### CODE REVIEW

File: _posts/2015-10-12-twis-37.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout:     post
+title:      This Week In Servo 37
+date:       2015-10-05 20:30:00
```
Comment (Manishearth) [inline _posts/2015-10-12-twis-37.md]: The date is wrong

**Comment by Manishearth** [comment]
r=me with the date fixed

File: _posts/2015-10-12-twis-37.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ - [Louis Chan](https://github.com/iawaknahc)
+ - [Pierre Chevalier](https://github.com/pierrechevalier83)
+ - [Qeole](https://github.com/Qeole)
+ - [Rahul Sharma](https://github.com/creativcoder)
+
+
+###Meetings
+
+At last week's [meeting](https://github.com/servo/servo/wiki/Meeting-2015-10-05), we discussed
+Browser.html, WebRender, Q4 planning, and our new hire, Alan Jeffrey!
```
Comment (Manishearth) [inline _posts/2015-10-12-twis-37.md:36]: Probably should mention that Alan joined the team up top

Comment (larsbergstrom) [inline _posts/2015-10-12-twis-37.md:36]: He just started this week, so I was going to save it for Monday :-)

Comment (Manishearth) [inline _posts/2015-10-12-twis-37.md:36]: makes sense

**Pull request merged**